src/common/components/layouts/pages/inspect/pdfDocument/PdfDocument.scss

```diff
@@ -35,3 +35,20 @@
     }
   }
 }
+
```

Comment:
I would rather prefer to place the alert at the center of the pdf viewer section. Let's try to wrap the viewer section and position the alert relative to the wrapper. It seems you won't need to change the position for small screens after that
![image](https://user-images.githubusercontent.com/31737885/191448585-816005c8-a5d7-4789-b5a8-c5b7d898945b.png)